1_data_structures_types/Task1.py

```diff
@@ -82,17 +99,31 @@ def do_bytes():
         print('I can only hold bytes!')
 
 
-    
+def do_stuff_with_sets():
+    basket = {'elden ring', 'dark souls', 'dark souls', 'mario', 'mario', 'sekiro'}
+    """sets are unordered with no duplicate elements"""
+    basket == {'elden ring', 'dark souls', 'mario', 'sekiro'}
+    letters = set('Aloha!')
+    'A' in letters
+    vowels = set()
+    vowels.add('a')
+    intersected = letters.intersection(vowels)
```

Comment:
yeah, `intersection` is a pretty common thing when using sets. we can also find diff between two sets by using `-` operator.

```
intersected = vowels - letters
``` 

can be pretty useful when validating some data:
```
required_keys = {'a', 'b', 'c'}
input_dict = {'a': 'AA', 'c': 'CC', 'd': 'DD'}
if missing_keys := required_keys - set(input_dict):
    print(Missing: {missing_keys}')
```